GANReviewTool/modules/GANReviewWikicodeGenerator.js

```diff
@@ -255,11 +256,12 @@ export class GANReviewWikicodeGenerator {
 	}
 
 	getFirstTemplateNameFromWikicode( wikicode ) {
-		const match = wikicode.match( /(?<=\{\{)[^|}]+/ );
-		if ( !match ) {
+		const templateFinder = new TemplateFinder( wikicode );
+		const template = templateFinder.firstTemplate();
+		if ( !template ) {
 			throw new Error( 'getFirstTemplateNameFromWikicode: No template found in Wikicode.' );
 		}
-		return match[ 0 ];
+		return TemplateFinder.removePrefix( template.name );
```

Comment:
This seems a bit verbose. Anything we can do to shorten the amount of code here? Maybe we can add a template.getNameWithoutPrefix() function or variable somewhere?